app/presenters/service_manual_service_standard_presenter.rb

```diff
@@ -0,0 +1,63 @@
+class ServiceManualServiceStandardPresenter
+  attr_reader :content_item
+
+  def initialize(content_item)
+    @content_item = content_item
+  end
+
+  def points
```

Comment:
I think the points attribute might be better in the model rather than a presenter. (we could also memoize the loaded and sorted list)